zygrader/config/versioning.py

```diff
@@ -97,6 +97,14 @@ def versioning_update_preferences():
     if compare_versions(version, user_version):
         config["spooky_mode"] = False
 
+    version = "5.0.0"
+    if compare_versions(version, user_version):
+        config["unicode_mode"] = False
+
+    version = "5.0.1"
+    if compare_versions(version, user_version):
+        config["theme"] = "Default"
+
```

Comment:
You're missing some changes from master here `update_config_with_new_preferences` which handles adding new preference keys automatically.